2-mandatory/3-stocks.js

```diff
@@ -64,7 +77,12 @@ function getPriceChanges(closingPricesForAllStocks) {
     The price should be shown with exactly 2 decimal places.
 */
 function highestPriceDescriptions(closingPricesForAllStocks, stocks) {
+
     // TODO
+    const priceDescription = [];
+    for (let i = 0; i < closingPricesForAllStocks.length; i++){
+        priceDescription.push(`The highest price of ${stocks[i].toUpperCase()} in the last 5 days was ${(((closingPricesForAllStocks[i]).sort(function(a,b){return a - b})[closingPricesForAllStocks[i].length-1]).toFixed(2))}`)    }
+    return priceDescription;
```

Comment:
This is great. The only suggestion I would make is to format your code so that it's easier to read. I think we are going to talk about this in class tomorrow :)